examples/patient-intake/README.md

```diff
@@ -62,3 +62,11 @@ Then, visit `http://localhost:7860/` in your browser to start a chatbot session.
 docker build -t chatbot .
 docker run --env-file .env -p 7860:7860 chatbot
 ```
+## Cartesia best practices
+
+Since this example is using cartesia checkout best practices given in cartesia page.LLM prompts should be modified accordingly.
```

Comment:
Thanks! Just a few nits.

```suggestion
Since this example is using Cartesia, checkout the best practices given in Cartesia's docs. LLM prompts should be modified accordingly.
```